docs/G--Model-System/statuses.md

```diff
@@ -0,0 +1,122 @@
+* `MID` idle - message is waiting to be sent.
+* `MSN` sent - the message was succesfully sent with no known error.
+* `MFL` failed - the system attempted to send the message, but there was an error.
+* `MRS` resend - the message will be resent.
+* `MRT` retry - the message was successfully generated, but the messenger was unable to send, but it can be retried.
+* `MDO` debug only - if debugging is on, a record is left; otherwise this message would have been deleted.
+
+See the class `EEM_Message` for more information.
+
+### Payments
+Payment objects are records of attempts to exchange funds. 
+Note: when registrants elect to pay with an offline payment methods (like Check and Invoice) no payment record is created until an event organizer manually records the payment was fulfilled.
+
+* `PFL` failed - there was a technical error while attempting payment. This is also the initial status of payments (which makes sense if there was a fatal error after the payment was created but before it was updated to something else).
+* `PPN` pending - The initial status while users are directed offsite for payment. Payments with this status are automatically updated to failed after they are expired
+* `PAP` approved - the funds were successfully received, and applied to the transaction.
+* `PDC` declined - the payment processor/gateway declined the user's card etc.
+* `PCN` cancelled - the payment was initiated (eg the user was sent off-site) but subsequently cancelled the payment
+
+### Registrations
+Registrations are reservations for places at the event. 
+
+* `RIC` incomplete -  Initial status for registrations when they are first created. Payments are NOT allowed. Automatically toggled to whatever the default Event registration status is upon completion of the attendee information reg step NO space reserved. Registration is NOT active
+* `RNA` not approved - Payments are NOT allowed. Event Admin must manually toggle STS_ID for it to change. No space reserved. Registration is active.
+* `RPP` pending payment - Payments are allowed and the status will automatically be toggled to `RAP` if payment is made in full by the attendee. No space reserved. Registration is active
+* `RWL` wait list - The registration has been placed on the waiting list (see the wait-list add-on). Payments are allowed. The status will automatically be toggled to RAP if payment is made in full by the attendee. No space reserved. Registration is active.
+* `RAP` approved - The registration has been approved (although the transaction may not yet be paid in full). Further payments are permitted if required. A space IS reserved. Registration is active.
+* `RCN` cancelled - The registration was cancelled. Payments are NOT allowed. NO space reserved. Registration is NOT active
+* `RDC` declined - The registration was the event manager. Payments are NOT allowed. No space reserved. Registration is NOT active
```

Comment:
> The registration was the event manager.

😕 sneaky event manager trying to get into their own event 😆 

maybe:

> The registration was cancelled by the event manager

and then for clarity, `RCN` cancelled could be changed to something like:

> The registration was cancelled by the registrant.

Reply:
:+1: thanks!